_includes/parse-server/file-adapters.md

```diff
@@ -175,6 +175,94 @@ var api = new ParseServer({
 
 Don't forget to change **S3_ACCESS_KEY**, **S3_SECRET_KEY** and **S3_BUCKET** to their correct value.
 
+##### S3Adapter configuration for Digital Ocean Spaces
+
+Spaces is an S3 equivalent prodivided by Digital Ocean. It's use the same api as S3 so you can use it with the S3 Adapter.
+You just need to change the AWS Endpoint to point to your Spaces endpoint.
+
+```javascript
+...
+var S3Adapter = require('parse-server').S3Adapter;
+var AWS = require("aws-sdk");
```

Comment:
Why do we need to require the AWS SDK?

Reply:
Oops. It turns out, you are right. I thought we need to process the 3rd-party provider's endpoint with `aws-sdk` to make it work. But it is working when I just pass it to adapter options as it is without the `aws-sdk`. Weird. Almost all examples on the internet have this aws-sdk thing. Even the README of [parse-s3-adapter](https://github.com/parse-community/parse-server-s3-adapter#usage-with-digital-ocean-spaces). Do you have any idea why everyone prefers to implement this through AWS SDK?

Reply:
It's not needed, its part of the adapter.